src/AzureDeprecation.APIs.REST/Mappings/DeprecationNoticeProfile.cs

```diff
@@ -0,0 +1,41 @@
+﻿using AutoMapper;
+using CodeJam.Strings;
+
+namespace AzureDeprecation.APIs.REST.Mappings;
+
+using Presentation = Contracts;
+using Db = DataAccess.Models;
+using Messages = AzureDeprecation.Contracts.Messages.v1;
+
+public class DeprecationNoticeProfile : Profile
+{
+    public DeprecationNoticeProfile()
+    {
+        CreateMap<Db.DeprecationNoticesResult, Presentation.DeprecationNoticesResponse>();
+        
+        CreateMap<Db.NoticeEntity, Presentation.DeprecationInfo>()
+            .IncludeMembers(i => i.DeprecationInfo)
+            .ForMember(d => d.Id, o => o.MapFrom(s => s.Id))
+            .ForMember(d => d.Links, o => 
+                o.MapFrom((x, _) =>
+                {
+                    if (x.PublishedNotice?.DashboardInfo?.Url.IsNullOrEmpty() ?? true)
+                        return new Dictionary<Presentation.ExternalLinkType, string?>();
```

Comment:
Aligning with current styling
```suggestion
                    if (x.PublishedNotice?.DashboardInfo?.Url.IsNullOrEmpty() ?? true)
                    {
                        return new Dictionary<Presentation.ExternalLinkType, string?>();
                    }
```

Also, shouldn't this be `== true` or is this some fancy new syntax?

Reply:
`== true`
It works either way, but I prefer your option